pkg/utils/environment_variable_test.go

```diff
@@ -159,7 +159,7 @@ func TestEnvironmentVariable(t *testing.T) {
 			expectedError := fmt.Errorf(
 				"CAKC010 Failed to parse %s. Reason: %s",
 				envVarName,
-				"time: invalid duration SOME_ENV_VAR_VALUE",
+				"time: invalid duration \"SOME_ENV_VAR_VALUE\"",
```

Comment:
```suggestion
				`time: invalid duration "SOME_ENV_VAR_VALUE"`,
```

Reply:
I think I'll keep this as-is, since it's consistent with another escaped quote in this file (and with other go tests we're running, for ex in the suite). thank you!